_src/edx-pl/sass/components/_colors.scss

```diff
@@ -61,13 +61,19 @@
 .swatch-meta {
     padding: spacing-vertical(xx-small) spacing-horizontal(small);
     white-space: nowrap;
-    font-size: font-size(small);
+    font-size: font-size(x-small);
     @include text-align(center);
     color: palette(grayscale, dark);
 
     .color-rgb {
```

Comment:
This is just normal behavior. The reason it works so nicely on Github is because there are no spaces. I'm going to try and remove the spaces between commas in our values. If I can get that working, it should do the trick.